RELEASES.md

```diff
@@ -129,45 +129,37 @@ deadline of 3 weeks.
   post.
 * Do a new PR to setup the next version
   * Update [VERSION.txt](VERSION.txt) to the next development release. E.g., "1.7.0-dev".
-  * `git mv docs/root/version_history/current.rst docs/root/version_history/v1.6.0.rst`, filling in the previous
+  * `git mv changelogs/current.yaml changelogs/1.6.0.yaml`, filling in the previous
     release version number in the filename and delete empty sections (like Incompatible Behavior Changes, Minor Bahavior Changes, etc).
     Add an entry for the new file in the `toctree` in
     [version_history.rst](docs/root/version_history/version_history.rst).
   * Edit the file you just created (eg `docs/root/version_history/v1.6.0.rst`) replacing the link part (between the `<>`) of any `ref:` links to point at the version - eg `` :ref:`Some link text <actual link>` `` -> `` :ref:`Some link text <v1.16:actual link>` ``
   * Create a new "current" version history file at the [release
-  notes](docs/root/version_history/current.rst) for the following version. E.g., "1.7.0 (pending)". Use
+  notes](changelogs/current.yaml) for the following version. E.g., "1.7.0 (pending)". Use
   this text as the template for the new file:
-```
-1.7.0 (Pending)
-===============
+```yaml
+date: Pending
+behavior_changes:
+// Changes that are expected to cause an incompatibility if applicable; deployment changes are likely required
 
-Incompatible Behavior Changes
------------------------------
-*Changes that are expected to cause an incompatibility if applicable; deployment changes are likely required*
+minor_behavior_changes:
+// Changes that may cause incompatibilities for some users, but should not for most*
 
-Minor Behavior Changes
-----------------------
-*Changes that may cause incompatibilities for some users, but should not for most*
+bug_fixes:
+// Changes expected to improve the state of the world and are unlikely to have negative effects*
 
-Bug Fixes
----------
-*Changes expected to improve the state of the world and are unlikely to have negative effects*
+removed_config_or_runtime:
+// Normally occurs at the end of the* :ref:`deprecation period <deprecated>`
```

Comment:
can you prefix these comments with `#` - im not sure that `//` is a valid yaml comment, but either way i think better use hashes

Reply:
Sure.